packages/react-native/update-android.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
```

Comment:
I always recommend `set -e` on shell scripts so that it exits on any error

Reply:
Added `set -e` to the top of the script.